index.html

```
@@ -1,12 +1,99 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="ko">
   <head>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </head> ], start tag match failed [ <head> ] on line 3.

(tag-pair)